### PR TITLE
Add-t-type-definition-to-all-the-Jorstad-CrossArea_Subclass-with-CxG-reference

### LINKED ISSUE #2593: Add t-type definition to all the Jorstad CrossArea_Subclass with CxG reference
```[tasklist]
### Tasks
- [x] [pvalb GABAergic cortical interneuron](http://purl.obolibrary.org/obo/CL_4023018)
- [x] [sst GABAergic cortical interneuron](http://purl.obolibrary.org/obo/CL_4023017)
- [x] [sst chodl GABAergic cortical interneuron](http://purl.obolibrary.org/obo/CL_4023121)
- [x] [lamp5 GABAergic cortical interneuron](http://purl.obolibrary.org/obo/CL_4023011)
- [ ] https://github.com/obophenotype/cell-ontology/issues/2595
- [x] [sncg GABAergic cortical interneuron](http://purl.obolibrary.org/obo/CL_4023015)
- [x] [VIP GABAergic cortical interneuron](http://purl.obolibrary.org/obo/CL_4023016)
- [x] [chandelier pvalb GABAergic cortical interneuron](http://purl.obolibrary.org/obo/CL_4023036)
- [x] [astrocyte of the cerebral cortex](http://purl.obolibrary.org/obo/CL_0002605)
- [x] [cerebral cortex endothelial cell](http://purl.obolibrary.org/obo/CL_1001602)
- [x] [microglia cell](http://purl.obolibrary.org/obo/CL_0000129)
- [x] [oligodendrocyte precursor cell](http://purl.obolibrary.org/obo/CL_0002453)
- [x] [vascular leptomeningeal cell](http://purl.obolibrary.org/obo/CL_4023051)
- [x] [L5 extratelencephalic projecting glutamatergic cortical neuron](http://purl.obolibrary.org/obo/CL_4023041)
- [x] [L5/6 near-projecting glutamatergic neuron](http://purl.obolibrary.org/obo/CL_4030067)
- [x] [L6 corticothalamic-projecting glutamatergic cortical neuron](http://purl.obolibrary.org/obo/CL_4023042)
- [x] [L6b glutamatergic cortical neuron](http://purl.obolibrary.org/obo/CL_4023038)
```

### PR DESCRIPTION
**Pull request by AvolaAmg**
Fixes #2593 #2343 #2333

### CODE REVIEW

**Comment by AvolaAmg** [comment]
have a look at the tickets above to track the changes.
In particular #2593 for the CL terms that have been modified.
and #2333 to check the changes done on sst GABAergic cortical interneuron.

**Comment by dosumis** [comment]
Sorry for confusion - but I think only justified for neurons  (see https://github.com/obophenotype/cell-ontology/issues/2333#issuecomment-2368000286).  I don't think there's any evidence of anything special about cortical endothelial cells or astrocytes.

File: src/ontology/cl-edit.owl [hunk 1]
```diff
@@ -28067,7 +28068,7 @@ EquivalentClasses(obo:CL_1001601 ObjectIntersectionOf(obo:CL_0000150 ObjectSomeV
 
 # Class: obo:CL_1001602 (cerebral cortex endothelial cell)
 
-AnnotationAssertion(Annotation(oboInOwl:hasDbXref "NPX:PDR") obo:IAO_0000115 obo:CL_1001602 "Endothelial cells forming the walls of the capillaries within the cerebral cortex.")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "NPX:PDR") Annotation(rdfs:comment "https://cellxgene.cziscience.com/e/5346f9c6-755e-4336-94cc-38706ec00c2f.cxg/") obo:IAO_0000115 obo:CL_1001602 "A transcriptomically distinct endothelial cell forming the walls of the capillaries within the cerebral cortex. The standard transcriptomic reference data for this cell type can be found on the CellxGene census under the collection: 'Transcriptomic cytoarchitecture reveals principles of human neocortex organization', dataset: 'Supercluster: Non-neuronal cells', Author Categories: 'CrossArea_subclass', clusters Endo.")
```
Comment (aleixpuigb) [inline src/ontology/cl-edit.owl:28071]: PR is failing because a dbxref is using the AP comment.

File: src/ontology/cl-edit.owl [hunk 2]
```diff
@@ -29552,7 +29557,7 @@ SubClassOf(obo:CL_4023036 ObjectSomeValuesFrom(obo:RO_0002292 obo:PR_000013502))
...
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "DOI:10.1101/2020.10.19.34312") Annotation(oboInOwl:hasDbXref "PMID:30382198") Annotation(oboInOwl:hasDbXref "PMID:37824655") Annotation(rdfs:comment "https://cellxgene.cziscience.com/e/d01c9dff-abd1-4825-bf30-2eb2ba74597e.cxg/") obo:IAO_0000115 obo:CL_4023038 "A glutamatergic neuron with a soma found in cortical layer 6b. They are transcriptomically related to corticothalamic-projecting neurons but have differential projections to the thalamus or anterior cingulate. The standard transcriptomic reference data for this cell type can be found on the CellxGene census under the collection: 'Transcriptomic cytoarchitecture reveals principles of human neocortex organization', dataset: 'Supercluster: Deep layer (non-IT) excitatory neuron', Author Categories: 'CrossArea_subclass', cluster L6b.")
```
Comment (aleixpuigb) [inline src/ontology/cl-edit.owl:29560]: PR is failing because a dbxref is using the AP comment

**Review by aleixpuigb — APPROVED** [review]
PR term will be imported when the imports are refreshed.

**Pull request merged**